docs/content/en/docs/workflows/dev.md

```diff
@@ -4,4 +4,44 @@ linkTitle: "Continuous development"
 weight: 1
 ---
 
-{{< todo 3074>}} dev page
\ No newline at end of file
+## `skaffold dev`
+
+`skaffold dev` enables continuous local development on an application.
+While in `dev` mode, Skaffold will watch an application's source files, and when it detects changes,
+will rebuild your images (or sync files to your running containers), push any new images, and redeploy the application to your cluster.
+
+`skaffold dev` is considered Skaffold's main mode of operation, as it allows you
+to leverage all of the main features of Skaffold in a continuous way while iterating
+on your application.
+
+
+## Dev loop
+
+When `skaffold dev` is run, Skaffold will first do a full build and deploy of all artifacts specified in the `skaffold.yaml`, identical behavior to `skaffold run`. Upon successful build and deploy, Skaffold will start watching all source file dependencies for all artifacts specified in the project. As changes are made to these source files, Skaffold will rebuild the associated artifacts, and redeploy the new changes to your cluster.
```

Comment:
I dont think we have `skaffold run` documented any where. we can drop it or link to existing CLI doc on run.